### ISSUE TITLE
Add the functionality for InterDimensional Communication

### ISSUE THREAD

**Issue by Keridos** [SlimeVoid/WirelessRedstone#23]
Hi there, i just forked your mod to see if I could add in Interdimensional communication, but it seems to me as if it is quite hard to do that since everything is bound to worlds.

We could aswell removed the getClosestTransmitter and let the communication follow a simple or-model like vanilla redstone does.

Refactoring the code might be something that is needed aswell. At least I found it quite hard to read in its current form.

**Comment by ali4z** [comment]
Everything is bound to worlds because the world is the scope. If anything the scope is too large.

If you have two worlds (IE, top and nether) you should be able to access them seperately if you have the world objects. So you can in theory route signals between them, via for example a inter-dimentional tx/rx bridge (listens/transmits on a frequency from another world).

getClosestTransmitter is used by triangulator, removing it is a terrible idea. It is not used to find transmitting nodes, as that would be stupid. The frequency state already follows an "or-model".

Why would re-factoring the code be needed? It is working and it has gone through years of optimizing. Redoing it would not only be a massive task but can easily break other parts of the mod. As it stands; re-doing the ether would me a gigantic waste of time, time which could be used doing something useful instead.

**Comment by Eurymachus** [comment]
I agree with Ali on this one. The Ether is working as intended and there is no need to alter it in anyway shape or form, it's highly efficient and performs as expected.

You wouldn't re-do the ISO Standard for wireless connectivity because you had a new kind of port, you would proxy between them, as Ali said an ether bridge and inter-dimensional transmitters and receivers. You would then store the respective link as data in the tile entity, you wouldn't rewrite the ether.

Inter-dimensional functionality would/will be an addon in the future. For now I'm
Personally migrating the current mods to 1.6.2

**Comment by Tarig0** [comment]
Hey Eury, how do you want to do this, Plugin or core and will we create a new block to be the bridge between dimensions or should it be an upgrade (possible to configure)

**Comment by Eurymachus** [comment]
Think I'd just like to add a new subsection like with camouflage.

However. Im not sure. It might be better just to add new blocks. Especially give the scope of blocks available to us in 1.6 and 1.7+

**Comment by Tarig0** [comment]
new block seems fine to me think we just need a interdim transmitter which will add it self as a source to the destination dim ID

**Comment by Tarig0** [comment]
That way recivers will act the same way
